### PR TITLE
Make ACLs for policies/policy_groups/cookbook_artifacts work

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

**Comment by jkeiser** [comment]
@chef/client-maintainers @danielsdeleo

File: spec/run_oc_pedant.rb [hunk 1]
```diff
@@ -142,7 +142,6 @@ def start_cheffs_server(chef_repo_path)
 
   Pedant.setup(pedant_args)
 
-  fail_fast = []
```
Comment (danielsdeleo) [inline spec/run_oc_pedant.rb]: I think you need to keep this.

**Comment by danielsdeleo** [comment]
LGTM, obviously needs some travis love.

File: Gemfile [hunk 2]
```diff
@@ -3,12 +3,9 @@ gemspec
...
-
-# gem 'oc-chef-pedant', :path => "../chef-server"
+gem 'oc-chef-pedant', :github => 'chef/chef-server', :branch => "jk/authorization-tags"
 
 # bundler resolve failure on "rspec_junit_formatter"
 # gem 'chef-pedant', :github => 'opscode/chef-pedant', :ref => "server-cli-option"
 
-gem 'chef', :github => 'chef/chef'
-# gem 'chef', :path => "../chef"
+gem 'chef', :github => 'chef/chef', :branch => 'jk/policies-acls'
```
Comment (randomcamel) [inline Gemfile]: Can we merge the other PRs first, so the Gemfile can go back to pointing at master?

Comment (randomcamel) [inline Gemfile]: Actually, never mind. Not that important, we're good at remembering these things now.

**Pull request merged**